src/main/java/persistence/sql/ddl/constraints/strategy/DefaultConstraintsStrategy.java

```diff
@@ -0,0 +1,22 @@
+package persistence.sql.ddl.constraints.strategy;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.stream.Collectors;
+import persistence.sql.ddl.constraints.builder.ConstraintsBuilder;
+import persistence.sql.ddl.constraints.builder.NotNullConstraintsBuilder;
+import persistence.sql.ddl.constraints.builder.UniqueConstraintsBuilder;
+
+public class DefaultConstraintsStrategy implements ConstraintsStrategy {
+    private final List<ConstraintsBuilder> constraintsBuilders = List.of(
+        new UniqueConstraintsBuilder(),
+        new NotNullConstraintsBuilder()
+    );
+
+    @Override
+    public String getConstraintsFrom(Field field) {
+        return constraintsBuilders.stream()
+            .map(constraintsBuilder -> constraintsBuilder.getConstraintsFrom(field))
+            .collect(Collectors.joining(" "));
+    }
```

Comment:
필드의 제약 조건들을 반영해주실 때 filter 를 사용해보는건 어떨까요?
어떤 이점을 얻을 수 있을 지도 고민해보시면 좋을 것 같습니다 😄 

```suggestion
        return constraintsBuilders.stream()
            .filter(constraintsBuilder -> constraintsBuilder.supports(field))        
            .map(constraintsBuilder -> constraintsBuilder.getConstraintsFrom(field))
            .collect(Collectors.joining(" "));
    }
```